### PR TITLE
add voluba v1 linear transform content type

### PR DESCRIPTION
**Pull request by xgui3783**
addresses https://github.com/HumanBrainProject/openMINDS_core/issues/350
edit: should also address https://github.com/FZJ-INM1-BDA/voluba/issues/132

cc @fsdavid @JThoennissen If you guys could raise any issue, if you can find any?

ping @lzehl Can you please review?

### CODE REVIEW

File: instances/data/contentTypes/voluba-v1-linearTransform.jsonld [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	"@context" : {
+		"@vocab": "https://openminds.ebrains.eu/vocab/"
+	},
+	"@id": "https://openminds.ebrains.eu/instances/contentTypes/application/vnd.voluba.v1.linearTransform+json",
+	"@type": "https://openminds.ebrains.eu/core/ContentType",
+	"fileExtension": [
+		".json"
+	],
+	"name": "application/vnd.voluba.v1.linearTransform+json",
+	"relatedMediaType": "https://www.iana.org/assignments/media-types/application/json",
```
Comment (lzehl) [inline instances/data/contentTypes/voluba-v1-linearTransform.jsonld]: this should only be filled if it is an exact match, correct @apdavison ? (or did we now do it also for just "relations"?)

File: instances/data/contentTypes/voluba-v1-linearTransform.jsonld [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	"synonym": [
+		"Voluba v1 Linear Transformation"
+	],
+    "specification": "https://raw.githubusercontent.com/FZJ-INM1-BDA/voluba/fa8e58dbe91a577dd0e6f6a8bf82c06b7e3e72e3/.openminds/voluba.v1.linearTransform.json"
```
Comment (lzehl) [inline instances/data/contentTypes/voluba-v1-linearTransform.jsonld]: this is not the specification :sweat_smile: that's just our content type registration. Leave it empty if you do not yet have a documentation on how this JSON looks like (required key-value pairs, expected values, etc.).

If you want to deliver at least a bit of documentation here use the "description" property to write a small (free text) paragraph directly in the content type instance.

**Review by lzehl — CHANGES_REQUESTED** [review]
@xgui3783 thank you for preparing this PR :slightly_smiling_face:  There is one issue that @apdavison should confirm/comment on and one smaller issue for the specification.

**Comment by xgui3783** [comment]
ping @lzehl, I have fixed the PR according to your suggestions.

**Comment by lzehl** [comment]
@xgui3783 I've updated the indentation to 2 spaces per tab (recent guideline for openMINDS) and updated the synonym a bit. If you are happy with it I can merge it.

**Comment by xgui3783** [comment]
@lzehl I am happy! Thanks for your help.

**Review by lzehl — APPROVED** [review]

**Pull request merged**